### ISSUE TITLE
List all units and constants in the docs

### ISSUE THREAD

**Issue by rafaqz** [PainterQubits/Unitful.jl#623]
Its not easy to find e.g. the Stefan Bolzmann constant. Searching the docs for `Stefan` returns nothing.

It would be nice if there was a page listing all the docs for all of these.

**Comment by ggggggggg** [comment]
Agreed, I just resorted to searching github to find the stephan boltzman constant, the boltzman constant and the charge of the electron. Maybe the constants could be re-exported into `Unitful.Constants` for easier tab completion as well?

**Comment by giordano** [comment]
Are you aware of [`PhysicalConstants.jl`](https://github.com/JuliaPhysics/PhysicalConstants.jl)?

**Comment by ggggggggg** [comment]
Yes, but there are already lots of useful constants in Unitful, I prefer to use those and have them easily accessible when it has what I need already.

**Comment by giordano** [comment]
Because I don't think constants should live here.
